kibble/api/languages.go

```diff
@@ -50,6 +51,17 @@ func loadAllLanguagesFromApi(cfg *models.Config, site *models.Site) error {
 	site.DefaultLanguage = formatPathLocale(languages.DefaultLanguage.Code)
 	site.Languages = languages.mapToModel()
 
+	log.Infof("Site Languages Loaded from API")
+	var langaugesList []string
+	for _, langs := range site.Languages {
+		if langs.Code != "" {
```

Comment:
This `if` is redundant... it's not possible to have an empty string for a language code... Uber wont allow it.

Reply:
in this case, the empty language code is not cause by the API, but by us change the `code `to be `""` for the default language, meaning it would print a blank entry, which ended up looking odd. I've updated the logic to be clearer about what it actually does, and also added the default language back into the Supported Languages list, which means that `if - nil - log` check wont be hit as frequently/ another reason to get rid of it

Reply:
Ah okay... lame lol